scripts/deploy_scale_dashboard.py

```diff
@@ -0,0 +1,246 @@
+        ,
+        {
+            "type": "metric",
+            "x": 6,
+            "y": 18,
+            "width": 6,
+            "height": 6,
+            "properties": {
+                "view": "timeSeries",
+                "metrics": [
+                    ["AWS/States", "LambdaFunctionsStarted", "LambdaFunctionArn",
+                     download_bundle_arn_prefix,
+                     {"stat": "Sum"}],
+                    [".", "LambdaFunctionsFailed", ".", ".", {"stat": "Sum"}],
+                    [".", "LambdaFunctionsSucceeded", ".", ".", {"stat": "Sum"}],
+                    [".", "LambdaFunctionsHeartbeatTimedOut", ".", ".", {"stat": "Sum"}]
+                ],
+                "region": region,
+                "stacked": False,
+                "title": "Download bundle",
+                "period": 300
+            }
+        },
+        {
+            "type": "metric",
+            "x": 12,
+            "y": 18,
+            "width": 6,
+            "height": 6,
+            "properties": {
+                "view": "timeSeries",
+                "metrics": [
+                    ["AWS/States", "LambdaFunctionsStarted", "LambdaFunctionArn",
+                     checkout_bundle_arn_prefix,
+                     {"stat": "Sum"}],
+                    [".", "LambdaFunctionsFailed", ".", ".", {"stat": "Sum"}],
+                    [".", "LambdaFunctionsSucceeded", ".", ".", {"stat": "Sum"}],
+                    [".", "LambdaFunctionsHeartbeatTimedOut", ".", ".", {"stat": "Sum"}]
+                ],
+                "region": region,
+                "stacked": False,
+                "title": "Checkout bundle",
+                "period": 300
+            }
+        },
+        {
+            "type": "metric",
+            "x": 0,
+            "y": 3,
+            "width": 24,
+            "height": 3,
+            "properties": {
+                "view": "timeSeries",
+                "stacked": False,
+                "metrics": get_metrics_array(upload_bundle_arn_prefix, 10),
+                "region": region,
+                "title": "Bundle upload runtime",
+                "period": 300
+            }
+        },
+        {
+            "type": "metric",
+            "x": 0,
+            "y": 6,
+            "width": 24,
+            "height": 3,
+            "properties": {
+                "view": "timeSeries",
+                "stacked": False,
+                "metrics": [
+                    ["AWS/States", "ExecutionTime", "StateMachineArn",
+                     "arn:aws:states:us-east-1:861229788715:stateMachine:dss-s3-copy-sfn-dev", {"period": 10}]
+                ],
+                "region": region,
+                "title": "Copy Execution Time",
+                "period": 300
+            }
+        }
+    ]
+}
+
+client = boto3.client('cloudwatch')
+
+response = client.put_dashboard(
+    DashboardName=f"Scaalbility-{stage}",
+    DashboardBody=json.dumps(dashboard_def)
+)
+
+print(str(response))
```

Comment:
May want to subset to just the relevant key or `del response["responseMetadata"]`

Reply:
returning just the status code now